test/integration/running/test_running_cluster.py

```diff
@@ -37,45 +37,58 @@ def stop_application(tt_cmd, app_name, workdir, instances):
         assert not os.path.exists(os.path.join(workdir, run_path, app_name, inst, "tarantool.pid"))
 
 
-@pytest.mark.skipif(tarantool_major_version < 3,
-                    reason="skip cluster instances test for Tarantool < 3")
-def test_running_base_functionality(tt_cmd, tmpdir_with_cfg):
-    tmpdir = tmpdir_with_cfg
-    app_name = "small_cluster_app"
+def wait_cluster_started(tt_cmd, workdir, app_name, instances, inst_conf):
+    files = []
+    for inst in instances:
+        conf = inst_conf(workdir, app_name, inst)
+        files.append(conf['pid_file'])
+        files.append(conf['console_socket'])
+    assert wait_files(5, files)
```

Comment:
We already have `wait_file` function in utils. Why can't we just reuse it in for loop (like it was before) instead of appending it in list and passing to the separate function?

Reply:
Because with `wait_file` we are not able to specify total timeout for waiting. I think here it's more important to set total timeout rather than timeout for every single instance.

Reply:
Maybe it's really important, but in this case it can create more flacky problems due to we will check for all instances while some of them are already running. As a result, the deadline will reach (because we have total timeout) and chance of deadline expiration is very high. Correct me, if I'm wrong.